scripts/benchmark_multithreaded.py

```diff
@@ -0,0 +1,190 @@
+# To prevent _tkinter.TclError: https://stackoverflow.com/a/37605654
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+import os
+import argparse
+import multiprocessing
+import subprocess
+import json
+
+from datetime import datetime
+
+
+
+BENCHMARK_EXECUTABLE = 'hyriseBenchmarkTPCH'
+DEFAULT_TPCH_QUERIES = [query for query in range(1, 23) if query != 15] # Exclude query 15 which is not supported in our multithreaded benchmarks
+MAX_CORE_COUNT = multiprocessing.cpu_count()
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--scale', action='store', type=float, metavar='S', default=0.1, help='TPC-H scale factor (default: 0.1)')
+    parser.add_argument('-v', '--verbose', action='store_true', help='Print log messages')
+    parser.add_argument('-q', '--queries', action='store', type=int, metavar='Q', nargs='+', help='Specify the TPC-H queries that will be benchmarked')
```

Comment:
To your first question: No, `--queries 1 3 5` is the way python argparse handles "list" parameters, comma-separated doesn't work.

Reply:
To your second question, that is unfortunately a `cxxopt` problem, which basically cannot handle "list" inputs like that. You'd have to use the `--queries` parameter for each query you want to add:
```
./build/hyriseBenchmarkTPCH --queries 1 --queries 2 -s 0.001
```
That's why I added the shortcut `-q` for `--queries` there, because it makes life much better when you instead can just do:
```
./build/hyriseBenchmarkTPCH -q 1 -q 2 -s 0.001
```

Reply:
Maybe accept a string in both cases and split it manually?